src/index.js

```diff
@@ -24,19 +24,7 @@ export type Props = {
   className: string,
 }
 
-type State = {
-  style: { [key: string]: any },
-  type: string,
-  speed: number,
-  width: number,
-  height: number,
-  primaryColor: string,
-  secondaryColor: string,
-  preserveAspectRatio: string,
-  className: string,
-}
-
-class ContentLoader extends React.Component<Props, State> {
+class ContentLoader extends React.Component<Props> {
   static defaultProps = {
```

Comment:
Since we'll now have a dumb component, we should have to set the `defaultProps` as another way: after the component and not inside it, like: 
```js
ContentLoader.defaultProps = {
    // object
}
```